llvm/test/CodeGen/AMDGPU/image-atomic-attributes.ll

```diff
@@ -0,0 +1,356 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-attributes --check-globals all --version 4
+; RUN: opt -S -mtriple=amdgcn-unknown-unknown < %s | FileCheck -check-prefixes=CHECK %s
```

Comment:
These tests don't show any transforms, and I assume you're just going for the direct test of the intrinsic attributes. If you're not going to show the effect on a transform of the attribute, this probably belongs in test/Assembler rather than test/CodeGen

Reply:
Attributes are checked at the end. Maybe that should be the only thing that is checked? I can remove the rest of the check lines in another PR. Sorry for the confusion.

Reply:
Yes, just check the attributes. And don't keep in CodeGen